crates/control/src/motion/command_sender.rs

```diff
@@ -0,0 +1,64 @@
+use color_eyre::{eyre::WrapErr, Result};
+use context_attribute::context;
+use framework::AdditionalOutput;
+use hardware::ActuatorInterface;
+use serde::{Deserialize, Serialize};
+use types::{led::Leds, motion_selection::MotionSafeExits, motor_commands::MotorCommand};
+
+#[derive(Deserialize, Serialize)]
+pub struct CommandSender {}
+
+#[context]
+pub struct CreationContext {}
+
+#[context]
+pub struct CycleContext {
+    motor_commands: Input<MotorCommand<f32>, "motor_commands">,
+    leds: Input<Leds, "leds">,
+
+    motion_safe_exits: CyclerState<MotionSafeExits, "motion_safe_exits">,
+    hardware_interface: HardwareInterface,
+
+    motion_safe_exits_output: AdditionalOutput<MotionSafeExits, "motion_safe_exits_output">,
+    executed_motor_commands: AdditionalOutput<MotorCommand<f32>, "executed_motor_commands">,
```

Comment:
Currently, this `AdditionalOutput` is not needed, as it is the exact same copy of `motor_commands`

Reply:
Yes, but here it is clearer that it is the executed command

Reply:
I disagree, the only clear one is the `MainOutput` if someone changes the commands sent to the joints one has to change this as well...

Reply:
Renamed as discussed